_sass/components/_components.Section.scss

```diff
@@ -14,49 +20,62 @@
       flex-wrap: wrap-reverse;
       justify-content: center;
     }
+
     .nav__profiles {
       position: relative;
       height: 30px;
       display: flex;
       align-items: center;
-    }
+      overflow-x: auto;
+      overflow-y: hidden;
 
-    span, 
-    a {
-      position: relative;
-      margin-left: 20px;
-      color: gray;
-      cursor: pointer;
-      position: relative;
-      font-weight: 400;
+      > div, > a {
+        &:not(:first-child) {
+          margin-left: 20px;
+        }
 
-      &::after {
-        content: '';
-        position: absolute;
-        bottom: -5px;
-        left: 0;
-        background-color: $primary;
-        opacity: 0;
-        height: 2px;
-        width: 100%;
-      }
+        position: relative;
+        color: gray;
+        cursor: pointer;
+        position: relative;
+        font-weight: 400;
 
-      &:hover, &:focus {
-        color: black;
-        outline: none;
-      }
-
-      &.active {
-        color: black;
-        
+        display: inline-block;
+        white-space: nowrap;
+        max-width: 250px;
+        text-overflow: ellipsis;
+        overflow: hidden;
+        flex-shrink: 0;
```

Comment:
250px was something I wrote - is it what you want?

Reply:
I don't think it matters that much in my opinion. The max characters the name's length can be 100. So, I think the 250px is long enough to display most of it